File: fourth-semester/programming-for-mobile-devices-001/class-002/exercise-001/MainActivity.java
package com.example.myapplication;

import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;

import android.view.View;
import android.widget.EditText;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        EditText TextInputEditText = findViewById(R.id.TextInputEditText);
        EditText TextInputEditText2 = findViewById(R.id.TextInputEditText2);
        EditText TextInputEditText3 = findViewById(R.id.TextInputEditText3);

        Button button = findViewById(R.id.button);

        button.setOnClickListener(new View.OnClickListener(){
            public void onClick(View v) {
                String name = TextInputEditText.getText().toString();
                String email = TextInputEditText2.getText().toString();
                String telephone = TextInputEditText3.getText().toString();
            }
        });
    }
}

File: fourth-semester/programming-for-mobile-devices-001/class-003/exercise-001/MainActivity.kt
package com.example.myapplication

import android.os.Bundle
import android.widget.Button
import android.widget.EditText
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(bundle: Bundle?) {
        super.onCreate(bundle)
        setContentView(R.layout.activity_main)

        val valueInput = findViewById<EditText>(R.id.valueInput)
        val exchangeValueInput = findViewById<EditText>(R.id.exchangeValueInput)
        val calculateButton = findViewById<Button>(R.id.calculateButton)
        val resultText = findViewById<TextView>(R.id.resultText)

        calculateButton.setOnClickListener {
            val value = valueInput.text.toString().toFloat()
            val exchangeValue = exchangeValueInput.text.toString().toFloat()
            val result = value * exchangeValue

            resultText.text = "Resultado: $result"
        }
    }
}